src/app-layer-parser.c

```diff
@@ -964,6 +964,8 @@ int AppLayerParserParse(ThreadVars *tv, AppLayerParserThreadCtx *alp_tctx, Flow
                    alstate, AppLayerGetProtoName(f->alproto));
     }
 
+    uint64_t p_tx_cnt = AppLayerParserGetTxCnt(f->proto, alproto, f->alstate);
```

Comment:
That would probably fix the segv in https://github.com/inliniac/suricata/pull/2274/commits/0a7adacbcad299a68f938bb975e8e2a9df6149e0